src/task.js

```diff
@@ -0,0 +1,33 @@
+// import data from './data.js';
+
+export default () => {
+  const data = [];
+
+  data.push({
+    id: 0,
+    completed: false,
+    description: 'list1',
+  },
+  {
+    id: 1,
+    completed: false,
+    description: 'list2',
+  },
+  {
+    id: 2,
+    completed: false,
+    description: 'list3',
+  });
+  const div = document.createElement('div');
+  const container = document.querySelector('.tasks');
+  div.classList.add('task-items');
+
+  let content = '';
+  data.forEach((d) => {
```

Comment:
- [ ] **Required:** You are doing a great job. However, you are required to _render the to-do list in their index, if your case ids._

**_Hint_** Consider `Array
    .sort((a, b) => a.id - b.id)`